### DISCUSSION TITLE
Write app with flutter

### DISCUSSION THREAD

**Discussion by Milad305** [vpnhood/VpnHood#400] [General]
Hello 
I dont have c# or .net skills but i have a question.
Is it possible to write client side with flutter if it possible explane me how . what ahould i do.
I want to get ios output with flutter if it is possible to write client with flutter

**Comment by trudyhood** [comment]
Sorry, we are not familiar with flutter. However, if it is a UI framework, you can use VpnHood Nugets and just focus on UI using that framework.

The following link may helpful for you 
https://docs.flutter.dev/get-started/flutter-for/xamarin-forms-devs
